docs/application/toc_all.md

```diff
@@ -72,67 +72,68 @@
-##### [Slider](/application/dotnet/guides/nui/nui-components/Slider.md)
-##### [Switch](/application/dotnet/guides/nui/nui-components/Switch.md)
-##### [Tab](/application/dotnet/guides/nui/nui-components/Tab.md)
-##### [Toast](/application/dotnet/guides/nui/nui-components/Toast.md)
-#### Animation
-##### [Overview](/application/dotnet/guides/nui/animation.md)
-##### [Animation Types](/application/dotnet/guides/nui/animation-types.md)
-#### [Resources](/application/dotnet/guides/nui/resources.md)
-#### [Palette](/application/dotnet/guides/nui/Palette.md)
-#### Window System
-##### [Overview](/application/dotnet/guides/nui/tizenshell.md)
-##### [Quickpanel Client](/application/dotnet/guides/nui/quickpanelclient.md)
-#### XAML
-##### [Overview](/application/dotnet/guides/nui/xaml/xaml-overview.md)
-##### [Get Started with NUI XAML](/application/dotnet/guides/nui/xaml/get-started-xaml.md)
-##### [XAML Support for Tizen.NUI](/application/dotnet/guides/nui/xaml/xaml-support-for-tizen-nui.md)
-##### [Define UI Components in XAML](/application/dotnet/guides/nui/xaml/ui-component-in-xaml.md)
-##### [Layout Define Example](/application/dotnet/guides/nui/xaml/layout-define-example.md)
-
-### OpenTK
-#### [Overview](/application/dotnet/guides/opentk/overview.md)
-#### [Quickstart](/application/dotnet/guides/opentk/quickstart.md)
-#### [OpenTK Window](/application/dotnet/guides/opentk/window-of-opentk.md)
-#### [Create Applications with SkiaSharp](/application/dotnet/guides/opentk/create-app-skiasharp.md)
-
-### Wearable Circular UI
-#### [Overview](/application/dotnet/guides/wcircularui/overview.md)
-#### [Quickstart](/application/dotnet/guides/wcircularui/quickstart.md)
-#### [Tizen Wearable XAML App Template](/application/dotnet/guides/wcircularui/quickstart_tizenxamlapptemplate.md)
+### User Interface
+#### NUI
+##### [Overview](/application/dotnet/guides/user-interface/nui/overview.md)
+##### [Layer](/application/dotnet/guides/user-interface/nui/layer.md)
+##### [Window](/application/dotnet/guides/user-interface/nui/window.md)
+##### Layouts
+###### [Overview](/application/dotnet/guides/user-interface/nui/layouts.md)
+###### [Linear Layout](/application/dotnet/guides/user-interface/nui/linear-layout.md)
+###### [Grid Layout](/application/dotnet/guides/user-interface/nui/grid-layout.md)
+###### [Flex Layout](/application/dotnet/guides/user-interface/nui/flex-layout.md)
+###### [Relative Layout](/application/dotnet/guides/user-interface/nui/relative-layout.md)
+###### [Create Custom Layout](/application/dotnet/guides/user-interface/nui/custom-layout.md)
+##### Components
+###### [Overview](/application/dotnet/guides/user-interface/nui/ui-components.md)
+###### [Styling UI Components](/application/dotnet/guides/user-interface/nui/styling-controls-with-xaml.md)
+###### [Visuals](/application/dotnet/guides/user-interface/nui/visuals.md)
+###### [View](/application/dotnet/guides/user-interface/nui/view.md)
```

Comment:
I have found **pagination.md** as one of the affected file due to **view.md** file relocation. 
I also noticed there is comment left (please see attached screenshot) in the **pagination.md** pointing to **view.md** file. Just wanted to check if would like to fix OR want to get this updated/removed in separate PR?

![PR#1491 - Query](https://user-images.githubusercontent.com/85727367/130575960-c1ec56f1-5cd1-4b03-a85a-f0af00879b63.PNG)